docs/content/tutorials/helm/index.md

```diff
@@ -0,0 +1,654 @@
+
+```txt
+> kubectl get all -n demo
+
+pod/redis-r5tcrra3d7uh6-7bcd8b8d8d-jmgn4   2/2     Running   0          51s
+pod/webapp-79d5dfb99-f6xgj                   1/1     Running   0          52s
+
+NAME                          TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)    AGE
+service/redis-r5tcrra3d7uh6   ClusterIP   10.43.104.63   <none>        6379/TCP   51s
+
+NAME                                  READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/redis-r5tcrra3d7uh6   1/1     1            1           51s
+deployment.apps/webapp                  1/1     1            1           52s
+
+NAME                                             DESIRED   CURRENT   READY   AGE
+replicaset.apps/redis-r5tcrra3d7uh6-7bcd8b8d8d   1         1         1       51s
+replicaset.apps/webapp-79d5dfb99                   1         1         1       52s
+
+NAME                TYPE                                  SECRET         STATUS
+recipe.radapp.io/db   Applications.Datastores/redisCaches   redis-secret   Ready
+```
+
+Look at the status of the `recipe.radapp.io/db` resource. If the status is not `Ready`, then try running the command again after a delay. The status should show as `Ready` when the Recipe has fully-deployed. You can also see additional resources starting with `redis-`. These were created by the Recipe.
+
+{{< alert title="⚠️ Missing resources" color="warning" >}}
+If you do not see the additional resources starting with `redis-` then it's likely they are in a different Kubernetes namespace. Run `kubectl get all -A` to see everything.
+{{< /alert >}}
+
+Now that you have added a Recipe, run this command to display the state of the Radius application:
+
+```bash
+rad app graph -a demo -g default-demo
+```
+
+The output should look like this:
+
+```txt
+> rad app graph -a demo -g default-demo
+Displaying application: demo
+
+Name: webapp (Applications.Core/containers)
+Connections: (none)
+Resources:
+  webapp (kubernetes: apps/Deployment)
+
+Name: db (Applications.Datastores/redisCaches)
+Connections: (none)
+Resources:
+  redis-r5tcrra3d7uh6 (kubernetes: apps/Deployment)
+  redis-r5tcrra3d7uh6 (kubernetes: core/Service)
+
+
+```
+
+`rad app graph` shows the **Application Graph** of the application. This includes:
+
+- Entries for each major resource: `webapp` is an `Applications.Core/containers` and `db` is an `Applications.Datastores/redisCaches`.
+- Connections between resources: (none yet, you will add this next).
+- Resources that were created: see the Kubernetes `Deployment` listed for `webapp` and the Kubernetes `Deployment` and `Service` listed for `db`.
+
+The Redis Cache created by the recipe is visible as part of the application. You can also see the `Resources` created in Kubernetes that make up the Redis Cache. In a previous step you saw these listed by `kubectl`. Since Radius deployed the Recipe its know that these resources *logically* are part of the Redis Cache in the application.
```

Comment:
```suggestion
The Redis Cache created by the recipe is visible as part of the application. You can also see the `Resources` created in Kubernetes that make up the Redis Cache. In a previous step you saw these listed by `kubectl`. Since Radius deployed the Recipe, it knows that these resources *logically* are part of the Redis Cache in the application.
```